package.json

```diff
@@ -11,7 +11,7 @@
   "version": "0.2.0-beta.8",
   "scripts": {
     "lint": "tslint 'src/**/*.ts'",
-    "build": "tsc && typedoc",
+    "build": "tsc && browserify -s Bitski -e dist/lib/bitski.js -t [ babelify --presets [ es2015 ] ] > dist/bitski.js && npm run docs",
```

Comment:
can we put some of this in a config?

Reply:
Would be nice, I didn't see anything about a config file like rollup has: https://github.com/browserify/browserify